ui/raidboss/data/07-dt/hunts/heritage_found.ts

```diff
@@ -0,0 +1,168 @@
+      response: Responses.getIn('info'),
+    },
+    {
+      id: 'Hunt Urna Magnetron',
+      type: 'StartsUsing',
+      netRegex: { id: '98C4', source: 'Urna Variabilis', capture: false },
+      response: Responses.aoe(),
+    },
+    {
+      id: 'Hunt Urna Thunderous Shower',
+      type: 'StartsUsing',
+      netRegex: { id: '98CB', source: 'Urna Variabilis' },
+      response: Responses.stackMarkerOn(),
+    },
+    {
+      id: 'Hunt Urna Electrowave',
+      type: 'StartsUsing',
+      netRegex: { id: '98CC', source: 'Urna Variabilis', capture: false },
+      response: Responses.aoe(),
+    },
+    {
+      id: 'Hunt Urna Magnetron Debuff',
+      type: 'GainsEffect',
+      // FE7: Positive Charge, FE8: Negative Charge
+      netRegex: { effectId: ['FE7', 'FE8'], source: 'Urna Variabilis' },
+      condition: Conditions.targetIsYou(),
+      run: (data, matches) =>
+        data.magnetronDebuff = matches.effectId === 'FE7' ? 'positive' : 'negative',
+    },
+    {
+      id: 'Hunt Urna Magnetoplasma',
+      type: 'StartsUsing',
+      // 98C5: Boss Positive, 98C7: Boss Negative
+      netRegex: { id: ['98C5', '98C7'], source: 'Urna Variabilis' },
+      durationSeconds: 8,
+      alertText: (data, matches, output) => {
+        const bossMagnet = matches.id === '98C5' ? 'positive' : 'negative';
+        const myMagnet = data.magnetronDebuff;
+        if (myMagnet === undefined)
+          return output.out!();
+
+        if (bossMagnet === myMagnet)
+          return output.combo!({ dir: output.out!(), magnet: output.repel!() });
+        return output.combo!({ dir: output.out!(), magnet: output.attract!() });
+      },
+      run: (data) => delete data.magnetronDebuff,
+      outputStrings: {
+        out: Outputs.out,
+        repel: {
+          en: 'Forced knockback',
+        },
+        attract: {
+          en: 'Forced draw-in',
+        },
+        combo: {
+          en: '${dir} (${magnet})',
+        },
+      },
+    },
+    {
+      id: 'Hunt Urna Magnetoring',
+      type: 'StartsUsing',
+      // 98C6: Boss Positive, 98C8: Boss Negative
+      netRegex: { id: ['98C6', '98C8'], source: 'Urna Variabilis' },
+      durationSeconds: 8,
+      alertText: (data, matches, output) => {
+        const bossMagnet = matches.id === '98C6' ? 'positive' : 'negative';
+        const myMagnet = data.magnetronDebuff;
+        if (myMagnet === undefined)
+          return output.in!();
+
+        if (bossMagnet === myMagnet)
+          return output.combo!({ dir: output.in!(), magnet: output.repel!() });
+        return output.combo!({ dir: output.in!(), magnet: output.attract!() });
+      },
+      run: (data) => delete data.magnetronDebuff,
+      outputStrings: {
+        in: Outputs.in,
+        repel: {
+          en: 'Forced knockback',
+        },
+        attract: {
+          en: 'Forced draw-in',
+        },
+        combo: {
+          en: '${dir} (${magnet})',
```

Comment:
Similar comment as above.  Since the forced movement happens before the boss aoe, I think this callout would be clearer as "Forced [knockback | draw-in], then in"